content/faq/docker/docker-machine.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Docker Machine
+showMetadata: true
+editable: true
+showToc: true
+---
+# The Docker Machine for AWS.
+
+Docker machine is a tool that lets you install Docker Engine on your virtual host or event Remote host at your cloud provider.
+
+By the way, This content will be talking about AWS cloud providers.
+
+It feels tedious when you want quickly spin up the prototype application using Docker. You might create and install tons of prerequisites. This thing is going to be fine if you have a lot of time. By the way, you don't have much time. What'd you like to do?
+
+Exactly, Docker Machine will help you a lot. So I will describe some tips starting from the beginning.
+
+If you want to know more about Docker-Machine and how does it work, please click this link.
+
+[https://docs.docker.com/machine/#what-is-docker-machine](https://docs.docker.com/machine/#what-is-docker-machine)
+
+In this article, I'd like to introduce you to the real-world usage of Docker Machine relies on the AWS EC2.
+
+First thing first, Install your docker machine to your machine. Please follow this link.
+
+[https://docs.docker.com/machine/install-machine/](https://docs.docker.com/machine/install-machine/)
+
+In the next step, Copy and Paste this command onto your terminal.
+
+```sh
+docker-machine create ${YOUR_MACHINE_NAME} \
+--driver amazonec2 \
+--amazonec2-region ap-southeast-1 \
+--amazonec2-open-port 27017 \
+--amazonec2-security-group security-my-awesome-ec2-project \
+--amazonec2-vpc-id \
+--amazonec2-subnet-id subnet-1 \
+--amazonec2-subnet-id subnet-2 \
+--amazonec2-private-address-only \
+--amazonec2-zone b \
+--amazonec2-zone c \
+--amazonec2-instance-type t3a.medium
```

Comment:
Does it support to specify zone or subnet-id many times, right? please clarify on this

Reply:
Sorry, My bad, After investigated, I found some interesting about multiple subnets. MrTrustor, He opens PR for a while. But, Unfortunately,  that PR was never merged to the master.
[https://github.com/docker/machine/pull/3740](https://github.com/docker/machine/pull/3740)

Thank you so much for pointing out my mistake. But the ironic thing is working when you put multiple subnets arguments.

Reply:
Is it possible for "--amazonec2-zone b --amazonec2-zone c"?